src/crate/client/http.py

```diff
@@ -118,12 +118,111 @@ def request(self,
         )
 
 
+def _json_loads_or_error(response):
```

Comment:
think the naming is a bit misleading, because `_json_loads` suggests to parse a string, but the method requires a response object